<ui version="4.0">
 <class>MainWidjet</class>
 <widget class="QWidget" name="MainWidjet">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>556</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>585</width>
    <height>556</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>600</width>
    <height>600</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::TabFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWidjet</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>2</y>
     <width>583</width>
     <height>41</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>41</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;Установка времени и даты &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="TimeEditWidget" native="true">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>60</y>
     <width>499</width>
     <height>245</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="SetTimeButton">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>508</y>
     <width>195</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Установить</string>
   </property>
  </widget>
  <widget class="QWidget" name="ResWidget" native="true">
   <property name="geometry">
    <rect>
     <x>42</x>
     <y>318</y>
     <width>499</width>
     <height>177</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>344</x>
     <y>508</y>
     <width>195</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
